test/openshift/e2e/sequential/1-018_validate_disable_default_instance/03-errors.yaml

```diff
@@ -0,0 +1,159 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: openshift-gitops
+  namespace: openshift-gitops
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: openshift-gitops-server
+  namespace: openshift-gitops
```

Comment:
@anandf I think we should check to make sure the other deployments and statefulset are also cleaned up while we are it

Reply:
Added deployments and statefulsets to the assertion.